test/plugins/individual-collateral/stargate/StargateRewardableWrapper.test.ts

```diff
@@ -255,10 +255,10 @@ describeFork('Wrapped S*USDC', () => {
     })
 
     it('claims previous rewards', async () => {
+      await wrapper.connect(bob).deposit(await mockPool.balanceOf(bob.address), bob.address)
```

Comment:
we should add a test for the new emergencyWithdraw flows to ensure correct behavior (funds are recoverable)

Reply:
Added the missing case!